tests/api/runtimes/test_nuclio.py

```diff
@@ -603,6 +603,21 @@ def test_deploy_with_security_context_enrichment(
             )
         )
 
+    def test_deploy_mlrun_requirements(
+        self, db: Session, k8s_secrets_mock: K8sSecretsMock
+    ):
+        user_unix_id = 1000
```

Comment:
I'm not sure if `user_unix_id` and `igz_version` are needed. see if you can remove them